packages/nitro-protocol/.env.example

```diff
@@ -2,4 +2,5 @@
 DEFAULT_GAS=6721975
 DEFAULT_GAS_PRICE=20000000000
 GANACHE_HOST=0.0.0.0
-GANACHE_PORT=8545
\ No newline at end of file
+GANACHE_PORT=8545
+NODE_ENV=dev
```

Comment:
If we're checking in `.env`, then I don't think we need a `.env.example` file at all.

Reply:
https://github.com/statechannels/monorepo/pull/424#discussion_r343954686